src/SignalR/common/Http.Connections/src/ConnectionsDependencyInjectionExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class ConnectionsDependencyInjectionExtensions
         public static IServiceCollection AddConnections(this IServiceCollection services)
         {
             services.AddRouting();
-            services.AddAuthorization();
+            services.AddAuthorizationPolicyEvaluator();
```

Comment:
hmmm. Seems like a lot of stuff was partially udpated.

Reply:
Yeah I'll double check things closer tomorrow, I just reverted the last PR and did a rename, appears that resulted in things in a weird state